src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -42,7 +42,17 @@ repos:
         - file
         - python
 {% endif %}
-{%- if cookiecutter.use_flake8 == "yes" %}
+
+{%- elif cookiecutter.auto_format_tool == 'ruff' +%}
+  - entry: ruff
+    id: ruff
+    name: ruff
+    exclude: |
+      ^{{cookiecutter.package_slug}}/scicookie/src/scicookie{{cookiecutter.project_slug}}/
```

Comment:
probably you will have some conflicts in this file. I just merged a refactoring PR that touches that file.

and probably it is wrong .. 
you probably should have just something like `exclude: ["tests"]`